.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 
 *            @chef/chef-workstation-reviewers @chef/chef-workstation-approvers @chef/chef-workstation-owners
 .expeditor/  @chef/jex-team
```

Comment:
@nikhil2611 feel free to remove this outdated team name

Reply:
Done